composer.json

```diff
@@ -0,0 +1,20 @@
+
+{
+    "name": "php/yaml",
```

Comment:
I'm just double-checking that the `php` vendor is not reserved on Packagist

Reply:
There is ongoing discussion about the top level namespace `php` being used on Packagist...

Reply:
see also php/pie#65

Reply:
Updated to use `pecl/yaml` package name